src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -199,12 +205,32 @@ class PublicationSerializer(serializers.Serializer):
             "GPP-Publicatiebank."
         )
     )
+    gepubliceerd_op = serializers.DateTimeField(
+        help_text=_(
+            "System timestamp reflecting when the publication was published in the "
+            "GPP-Publicatiebank."
+        )
+    )
```

Comment:
This should also get `required=False` and `allow_null=True` to be backwards compatible.